pages/linux/cmus.md

```diff
@@ -0,0 +1,24 @@
+# cmus
+
+> Commandline C* Music Player.
```

Comment:
C* comes from the package description.
It was written in C and C++, and I think the * is being used as a regex-style wildcard.